### ISSUE TITLE
Warning on MacOS: Secure coding is automatically enabled for restorable state

### ISSUE THREAD

**Issue by leuchtum** [jgraph/drawio-desktop#1740]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

You must agree to search and the code of conduct. You must fill in this entire template. If you delete part/all or miss parts out your issue will be closed.

If you are technical, you should reporting bugs along the lines of https://marker.io/blog/how-to-write-bug-report. If you are not technical, we will make allowances, please try to make an effort to understand the process.

**Describe the bug**
When invoking drawio from the CLI on MacOS, an WARNING is thrown. I'm not quite sure where it's coming from, but it's Apple related for sure. For example se https://github.com/python/cpython/issues/110950 

**To Reproduce**
1. Only MacOS
2. When ever invoking drawio from the CLI, following warning is shown:
```shell
2024-05-16 12:19:42.855 draw.io[95937:13418976] WARNING: Secure coding is automatically enabled for restorable state! However, not on all supported macOS versions of this application. Opt-in to secure coding explicitly by implementing NSApplicationDelegate.applicationSupportsSecureRestorableState:.
```
3. For example the version:

```shell
> /Applications/draw.io.app/Contents/MacOS/draw.io --version
2024-05-16 12:19:42.855 draw.io[95937:13418976] WARNING: Secure coding is automatically enabled for restorable state! However, not on all supported macOS versions of this application. Opt-in to secure coding explicitly by implementing NSApplicationDelegate.applicationSupportsSecureRestorableState:.
24.4.0
```

4. Or some export

```shell
drawio -x -f png does-not-exist.drawio
2024-05-16 13:18:43.217 draw.io[5251:13472341] WARNING: Secure coding is automatically enabled for restorable state! However, not on all supported macOS versions of this application. Opt-in to secure coding explicitly by implementing NSApplicationDelegate.applicationSupportsSecureRestorableState:.
Error: input file/directory not found
```

**Expected behavior**
NA

**Screenshots**
NA

**draw.io version (In the Help->About menu of the draw.io editor):**

```shell
> /Applications/draw.io.app/Contents/MacOS/draw.io --version
2024-05-16 12:19:42.855 draw.io[95937:13418976] WARNING: Secure coding is automatically enabled for restorable state! However, not on all supported macOS versions of this application. Opt-in to secure coding explicitly by implementing NSApplicationDelegate.applicationSupportsSecureRestorableState:.
24.4.0
```

**Desktop (please complete the following information):**

```shell
> sw_vers
ProductName:		macOS
ProductVersion:		14.4.1
BuildVersion:		23E224
```

**Smartphone (please complete the following information):**
NA

**I tested the problem in incognito/private mode with all browser extensions switched off, write "yes" below:**
NA

**Additional context**

**Comment by leuchtum** [comment]
Installed via homebrew.

**Comment by davidjgraph** [comment]
First questions are who added it to homebrew, what did they add and how did they add it?

**Comment by davidjgraph** [comment]
Looks like https://formulae.brew.sh/api/cask/drawio.json.

**Comment by leuchtum** [comment]
I've uninstalled the homebrew version and directly installed the releases published on github. On all versions 20.7.4, 24.2.5 and 24.4.0 this issue is present.

**Comment by davidjgraph** [comment]
Does this still happen in 24.4.6?

**Comment by leuchtum** [comment]
Resolved with 24.4.6, thanks
